test/code/plugins/oms_configuration_test.rb

```diff
@@ -151,6 +151,7 @@ def test_load_configuration()
       assert_equal(TEST_UUID, Configuration.uuid, "VM UUID should be loaded")
       assert_not_equal(nil, Configuration.cert, "Certificate should be loaded")
       assert_not_equal(nil, Configuration.key, "Key should be loaded")
+      p "get_azure_region_from_imds=", OMS::Configuration.get_azure_region_from_imds()
```

Comment:
Why are we doing a print here instead of assert check?

Reply:
testing to fix the UT